ResourceStatusSystem/Client/ResourceStatusClient.py

```diff
@@ -46,6 +46,56 @@ def _prepare(self, sendDict):
 
     return sendDict
 
+  def insert(self, tableName, record):
+    """
+    Insert a dictionary `record` as a row in table `tableName`
+
+    :params:
```

Comment:
Just do:
```
:param str tableName: the name of the table
:param dict record: dictionary of records to insert in the table
```

and also similarlly below.

Reply:
Done